lib/actions/ui.js

```diff
@@ -121,36 +123,36 @@ export function moveRight () {
   };
 }
 
-export function moveTo (i) {
+export function moveTo(i) {
   return (dispatch, getState) => {
     dispatch({
       type: UI_MOVE_TO,
       index: i,
-      effect () {
-        const { sessions } = getState();
+      effect() {
+        const {sessions} = getState();
         const uid = sessions.activeUid;
         const sessionUids = keys(sessions.sessions);
         if (uid === sessionUids[i]) {
           console.log('ignoring same uid');
-        } else if (null != sessionUids[i]) {
```

Comment:
`null != sessionUids[i]` checks for `undefined` as well @matheuss, now you're explicitly checking for the value `null`